app/controllers/user_projects_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UserProjectsController < ApplicationController
```

Comment:
In this controller you should create an action that will assign ANY user to the user_project not current user, current user is assigned when he creates the project(automatically becomes its contributor)